shared/message/input/AudioData.proto

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2013 NUbots <nubots@nubots.net>
+ */
+
+syntax = "proto3";
+
+package message.input;
+
+import "google/protobuf/timestamp.proto";
+
+message AudioData{
+        bytes WavData = 1;
```

Comment:
```suggestion
        bytes audio = 1;
```
or something similar. This is not specifically data from a wav file, it is any audio data, ie the message will also be used with a microphone module. should also add in sample rate.

Reply:
Another thing is if you ever want to be able to use this in realtime you'll need to find a way to make this be chunked, If I'm reading this right currently this always just contains the entire WAV file